### ISSUE TITLE
Fare_media and transaction_type overlap

### ISSUE THREAD

**Issue by antrim** [ODOT-PTS/GTFS-ride#23]
There is overlap in the **rider_trip.transaction_type** and **rider_trip.fare_media** fields. i.e. both list cash, transfer options (though slightly differently). If a traveler used a mobile payment "software flash pass" to make a transfer, then would transaction_type = 2 and fare_media = 4? It seems as though options 0, 1, 2 might be removed from transaction_type.

Quoting draft:

> transaction_type | Optional | The transaction_type field indicates what entitled the customer to the trip.
> -- | -- | --
>   |   | * 0 - customer paid cash/credit/debit.
>   |   | * 1 - customer used stored value or tokens.
>   |   | * 2 - customer used a transfer.
>   |   | * 3 - customer used a pass.
>   |   | * 4 - customer used a promotional coupon.
>   |   | * 5 - trip is a free trip.
>   |   | * 6 - customer was never charged the fare.
>   |   | * 7 - customer was charged a fare but did not pay.
>   |   | * 8 - Other.
> fare_media | Optional | The fare_media field indicates what media was used to pay the fare.
>   |   | * 0 - Not applicable or unknown.
>   |   | * 1 - Cash.
>   |   | * 2 - Paper transfer, single-use paper ticket, or token.
>   |   | * 3 - Paper flash pass (visual inspection).
>   |   | * 4 - Software flash pass (visual inspection).
>   |   | * 5 - Proof-of-payment receipt
>   |   | * 6 - Magnetic strip card, agency-issued.
>   |   | * 7 - RFID or smart card, agency-issued
>   |   | * 8 - Magnetic strip card, open payment.
>   |   | * 9 - RFID or smart card, open payment

**Comment by antrim** [comment]
Also, see issue #22, "ODX - transfers / multiple vehicle trips":

If GTFS-ride is intended to include linked trips (ODX), it may be useful to include these fields in **rider_trip_segment.txt** so that they could be associated with alightings (necessary in the case of transfers).

**Comment by carletop** [comment]
I agree that there is overlap here and perhaps some consolidation/clarification may be warranted to disentangle the intent and use of the two fields. The field values you suggested for the example would be correct. I will work on a pull request for this.

**Comment by antrim** [comment]
Why are RFID, smartphone apps, and open payment are listed across these definitions for `fare_media`?

> * 7 - RFID, smart card, or smartphone app., agency-issued.
> * 8 - Magnetic strip card, open payment.
> * 9 - RFID, smart card, or smartphone app., open payment.
